types/body-scroll-lock/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for body-scroll-lock 2.6
+// Project: https://github.com/willmcpo/body-scroll-lock (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: kimcoder <https://github.com/kimcoder>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```

Comment:
From the npmjs page for this library, it seems like this is a UMD module.
If so, you should add the following line:
```export as namespace bodyScrollLock;```